Rubberduck.Resources/Inspections/InspectionResults.resx

```diff
@@ -352,4 +352,8 @@
   <data name="SheetAccessedUsingStringInspection" xml:space="preserve">
     <value>Statically accessible sheet can be referred to by its code name</value>
   </data>
+  <data name="ObsoleteMemberUsageInspection" xml:space="preserve">
+    <value>Consider replacing the call to '{0}' {1}</value>
```

Comment:
How about just this? `Member '{0}' is marked as obsolete. {1}` - that way punctuation isn't missing if there's no documentation string for it?